application/routes/web.php

```diff
@@ -37,18 +37,31 @@
 
 Route::middleware(['auth', 'verified'])->group(function () {
     // Ballot
-    Route::get('/ballots/create', [BallotController::class, 'create'])->name('ballots.create');
-    Route::get('/ballots/{ballot:id}', [BallotController::class, 'view'])->name('ballots.view');
-    Route::get('/ballots/{ballot}/edit', [BallotController::class, 'edit'])->name('ballots.edit');
+    Route::prefix('/ballots')->as('ballots.')->group(function () {
+        Route::get('/', [BallotController::class, 'index'])->name('index');
 
-    Route::post('/ballots/create', [BallotController::class, 'store'])->name('ballots.store');
-    Route::patch('/ballots/{ballot}/update', [BallotController::class, 'update'])->name('ballots.update');
-    Route::delete('/ballots/{ballot}/delete', [BallotController::class, 'destroy'])->name('ballots.destroy');
+        Route::get('/create', [BallotController::class, 'create'])->name('create');
```

Comment:
I like how you have grouped related routes using the `->group()` with `prefix`
But the rout name prefix you had before e.g. `ballots.store` is now simply `create` - isn't this a bit vague?

Wouldn't you run into naming conflict as you develop out further routes, which will also has `create` method?